plugins/Ntuplizer.cc

```diff
@@ -1778,6 +1876,7 @@ Ntuplizer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
     nCHSFatJets = CHSFatJetsVect.size();
 
     // TODO: Move this calculation to the PFCandidate section
+    if (!isShort){
```

Comment:
Fat jets are currently not used for tracking either. Make sure this is not unnecessarily run!

Reply:
This is actually not needed. As long as `WriteFatJets = False`, the `CHSFatJetsVect` is empy